common/metrics/metrics.go

```diff
@@ -28,6 +28,8 @@ func GetMigrationData(conv *internal.Conv, driver, targetDb, typeOfConv string)
 		migrationData.TargetDialect = migration.MigrationData_GOOGLE_STANDARD_SQL.Enum()
 	case constants.TargetExperimentalPostgres:
 		migrationData.TargetDialect = migration.MigrationData_POSTGRESQL_STANDARD_SQL.Enum()
+	default:
```

Comment:
The default should be to use Google standard if nothing is provided. If an unrecognized string is provided, then we should throw an error. Is this being followed in this flow ?

Reply:
The default dialect is set as Google standard in case the dialect is not specified as postgres : https://github.com/cloudspannerecosystem/harbourbridge/blob/ce1ab31246486afa51660f7349eb3cc7e3035cd8/common/utils/utils.go#L489 . Should we handle unknown strings here instead?